lib/packages/core/src/typings/account.ts

```diff
@@ -7,44 +7,55 @@
 
 import { Keys } from '@burstjs/crypto';
 import { Transaction } from './transaction';
+import { AssetBalance } from './assetBalance';
+import { UnconfirmedAssetBalance } from './unconfirmedAssetBalance';
 
 /**
 * Account class
 *
 * The account class serves as a model for a Burstcoin account.
-* Each account contains its Burst address and numeric id.
+* It's meant to model the response from BRS API, except publicKey
+* has been moved into the keys object.
 */
 export class Account {
```

Comment:
Better would be pure interface.  The constructor is full of boilerplate

Reply:
Agreed, would be a bigger change but the right move.